benchmarks/BenchmarkPlugin.cpp

```diff
@@ -0,0 +1,57 @@
+//////////////////////////////////////////////////////////////////////////////////////
```

Comment:
I think we can remove this file since BenchmarkForall.cpp already covers the same functionality (albeit through the macro layer).